impeller/docs/standalone_gles.md

```diff
@@ -0,0 +1,195 @@
+# Using Impeller as a Standalone Rendering Library (with OpenGL ES)
+
+This guide describes how to use Impeller as a standalone rendering library using OpenGL ES. Additionally, some form of WSI (Window System Integration) is essential. Since EGL is the most the most popular form of performing WSI on platforms with OpenGL ES, Impeller has a toolkit that assists in working with EGL. This guide will use that toolkit.
```

Comment:
this document uses "OpenGL ES" and "EGL" but it's not immediately clear what is meant by each, might be worth adding a brief sentence clarifying that at the top.

Reply:
Will add these to the glossary and xref here. https://github.com/flutter/flutter/issues/107609#issuecomment-2097066395